src/test/java/com/google/api/generator/spring/composer/goldens/EchoSpringPropertiesFull.golden

```diff
@@ -36,55 +36,17 @@ public class EchoSpringProperties implements CredentialsSupplier {
 
   private String quotaProjectId;
   private Integer executorThreadCount;
-  private Duration echoInitialRetryDelay;
-  private Double echoRetryDelayMultiplier;
-  private Duration echoMaxRetryDelay;
-  private Duration echoInitialRpcTimeout;
-  private Double echoRpcTimeoutMultiplier;
-  private Duration echoMaxRpcTimeout;
-  private Duration echoTotalTimeout;
-  private Duration expandInitialRetryDelay;
-  private Double expandRetryDelayMultiplier;
-  private Duration expandMaxRetryDelay;
-  private Duration expandInitialRpcTimeout;
-  private Double expandRpcTimeoutMultiplier;
-  private Duration expandMaxRpcTimeout;
-  private Duration expandTotalTimeout;
-  private Duration collectInitialRpcTimeout;
-  private Double collectRpcTimeoutMultiplier;
-  private Duration collectMaxRpcTimeout;
-  private Duration collectTotalTimeout;
-  private Duration chatInitialRpcTimeout;
-  private Double chatRpcTimeoutMultiplier;
-  private Duration chatMaxRpcTimeout;
-  private Duration chatTotalTimeout;
-  private Duration chatAgainInitialRpcTimeout;
-  private Double chatAgainRpcTimeoutMultiplier;
-  private Duration chatAgainMaxRpcTimeout;
-  private Duration chatAgainTotalTimeout;
-  private Duration pagedExpandInitialRetryDelay;
-  private Double pagedExpandRetryDelayMultiplier;
-  private Duration pagedExpandMaxRetryDelay;
-  private Duration pagedExpandInitialRpcTimeout;
-  private Double pagedExpandRpcTimeoutMultiplier;
-  private Duration pagedExpandMaxRpcTimeout;
-  private Duration pagedExpandTotalTimeout;
-  private Duration simplePagedExpandInitialRpcTimeout;
-  private Double simplePagedExpandRpcTimeoutMultiplier;
-  private Duration simplePagedExpandMaxRpcTimeout;
-  private Duration simplePagedExpandTotalTimeout;
-  private Duration waitInitialRpcTimeout;
-  private Double waitRpcTimeoutMultiplier;
-  private Duration waitMaxRpcTimeout;
-  private Duration waitTotalTimeout;
-  private Duration blockInitialRpcTimeout;
-  private Double blockRpcTimeoutMultiplier;
-  private Duration blockMaxRpcTimeout;
-  private Duration blockTotalTimeout;
-  private Duration collideNameInitialRpcTimeout;
-  private Double collideNameRpcTimeoutMultiplier;
-  private Duration collideNameMaxRpcTimeout;
-  private Duration collideNameTotalTimeout;
+  private Retry retrySettings;
```

Comment:
Do we need `NestedConfigurationProperty` on each of them?

Reply:
That's a good point! I missed this since it didn't directly affect the properties binding, but will add to each field.